tests/blobstore_common_tests.py

```diff
@@ -89,23 +100,21 @@ def testListV2(self):
         """
         Ensure that the ```list_v2``` method returns sane data.
         """
-        items = list(self.handle.list_v2(
+        keys = list((key for key, item in self.handle.list_v2(
             self.test_fixtures_bucket,
             "test_good_source_data/0",
-        ))
-        self.assertIn("test_good_source_data/0", items)
-        for item in items:
-            if item == "test_good_source_data/0":
-                break
-        else:
-            self.fail("did not find the requisite key")
+        )))
+        self.assertIn("test_good_source_data/0", keys)
 
         # fetch a bunch of items all at once.
         items = list(self.handle.list_v2(
             self.test_fixtures_bucket,
             "testList/prefix",
         ))
         self.assertEqual(len(items), 10)
+        for ix, (key, metadata) in enumerate(items):
+            self.assertTrue(f"prefix.00{ix}" in key)
+            self.assertTrue(all(metadata_key in metadata for metadata_key in BlobMetadataField))
```

Comment:
do you care if the reverse is true, i.e., all the keys in BlobMetadataField exist in each key's metadata?